ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/ExifReader.java

```diff
@@ -25,7 +25,7 @@
     }
 
     @NonNull
-    public String getUserComment() {
+    String getUserComment() {
         try {
             final JpegImageMetadata jpegMetadata = (JpegImageMetadata) getMetadata(jpeg);
```

Comment:
Why holds the ExifReader the jpeg byte array? Would it be better to have a separate ExifReader factory method, which extracts the JpegImageMetadata and creates a new ExifReader from it which could read the separate fields?